src/main/java/uk/gov/justice/hmpps/prison/util/ResourceUtils.java

```diff
@@ -1,9 +1,18 @@
 package uk.gov.justice.hmpps.prison.util;
 
+import org.apache.commons.lang3.StringUtils;
+
 public enum ResourceUtils {
     INSTANCE;
 
     public static <T> T nvl(final T assertNullVal, final T defaultVal) {
         return assertNullVal != null ? assertNullVal : defaultVal;
     }
+
+    public static String getUniqueClientId(final String clientName, final String clientUniqueRef) {
+        if (StringUtils.isBlank(clientUniqueRef)) {
```

Comment:
Is this required - already tested in the API.  If so, better to throw another `ValidationException`?

Reply:
well this is a refactor from `NomisApiV1ResourceImpl`.  In `NomisApiV1Resource.getHolds` it is optional so can be null, which is therefore this scenario